### DISCUSSION TITLE
New user, hello!

### DISCUSSION THREAD

**Discussion by silvaf6** [fast4x/RiMusic#281] [Q&A]
There are some open source similar apps, but this one seems like the one that is the most functional and complete!

I just have one question, when on mobile data what audio quality does it use? When i download my playlists, are they always on higher quality possible as in youtube music?

Just one bug on my fold 4 one ui 6 if i can report here. It only downloads on background if i leave the player on the taskbar. Ive also disabled battery optimization and putted the tab open all the time while it downloads. 

Thanks alot!

**Comment by fast4x** [comment]
Hello, in version 0.6.14 the quality is always maximum. From 0.6.15 you can set it but it will only be used if there is a version of the song for that quality. Otherwise the upper one is downloaded.

For the bug, I don’t think it’s a bug, it looks like a background permission issue on device.

Thank you for your feedback on RiMusic.
